src/tests/dbus-tests/test_10_basic.py

```diff
@@ -11,11 +12,8 @@ def setUp(self):
         self.manager_obj = self.get_object('/Manager')
 
     def _get_modules(self):
-        content = self.read_file('/etc/os-release')
-        release = {key: value for (key, value) in [line.split('=') for line in content.split('\n') if line]}
-        distro = release['ID'].replace('"', '')
-
-        if distro in ('redhat', 'centos'):
+        distro, version, name = platform.linux_distribution()
```

Comment:
Please don't use `platform.linux_distribution` -- it's deprecated and will be removed in Python 3.7 -- https://docs.python.org/3/library/platform.html#platform.linux_distribution